docs/source/getting-started/index.rst

```diff
@@ -1,38 +1,66 @@
+ScanCode Workbench-ScanCode Toolkit Compatibility
+=================================================
 
--  Once downloaded, you can find ScanCode-Workbench under
-   ``dist/ScanCode-Workbench-<os>-x64-<version>``.
+-  ScanCode Workbench >= v3.1.1 is only compatible with scans from ScanCode v3.1.1 and above
+   that have been run with the ScanCode Toolkit ``-i`` option.
 
--  ScanCode Workbench >= v2 is only compatible with scans from ScanCode v2.0.0 and above
-   which are run with the ScanCode ``-i`` option. (For a list of available ScanCode Toolkit options
-   see the ScanCode Toolkit documentation:
-   :doc:`scancode-toolkit:tutorials/how_to_set_what_will_be_detected_in_a_scan`.
+   -  A list of available ScanCode Toolkit options is available in the ScanCode Toolkit
+      documentation:
+      :doc:`scancode-toolkit:tutorials/how_to_set_what_will_be_detected_in_a_scan`.
 
-Import a Sample Scan
-====================
+Open ScanCode Workbench and Load a ScanCode Toolkit Scan
+========================================================
+
+-  Double-click the ScanCode Workbench executable you downloaded.  You'll probably want to
+   maximize the application once it has opened.
+
+-  Import your JSON scan file and save it as a SQLite file (ScanCode Workbench works with the
+   data in a SQLite database).
+
+   -  :kbd:`File` > :kbd:`Import JSON File` (:kbd:`Ctrl` + :kbd:`I`) ==> opens
+      ``Open a JSON File`` window.
 
-We have provided a set of sample scans that you can quickly review in ScanCode Workbench in
-order to get a sense of its functionality and the types of information captured by a scan.
+   -  Select your JSON scan and click :kbd:`Open` ==> opens ``Save a SQLite Database File`` window.
 
-The samples are located at https://github.com/nexB/scancode-workbench/tree/develop/samples.
+   -  Keep or modify the default SQLite filename and click :kbd:`Save`.
 
-Import a ScanCode Toolkit Scan of Your Own Code
-===============================================
+-  You're now looking at your scan data displayed in the Table View -- the Table View itself is on
+   the right, and the Directory Tree (which is visible in all views) is on the left.
+
+   .. figure:: data/initial_load_getting_started.png
+      :class: with-border
+      :width: 600px
+
+      ..
+
+      Your first imported ScanCode Toolkit Scan.
+
+You can find additional details in the :ref:`how-to-guides` section below.
+
+Try a Sample Scan
+====================
 
-Import a ScanCode JSON file, and see what components are in your software!
+We've also provided a set of `sample scans <https://github.com/nexB/scancode-workbench/tree/develop/samples>`__
+that you can  review in ScanCode Workbench in order to get a sense of its functionality and the
+types of information captured by a scan.
 
-If you're ready, keep on reading this documentation for more information on how to use
-ScanCode Workbench.
+:guilabel:`[The samples need to be updated from 2.9.2 to the current version of SCTK.]`
```

Comment:
Yes they do ;) Can you enter a ticket for this (if we do not already have one). It doesnt need to be super detailed, just this sentence.

Reply:
Thanks @majurg -- I'll assign to myself and try to wrap up today or tomorrow.